### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by PeteUgwu**
## ##Popup Window

- [ ] Fix linters issues
- [ ] Update READme file
- [ ] Used basic JavaScript syntax.
- [ ] Used JavaScript to manipulate DOM elements.
- [ ] Used JavaScript events.
- [ ] Used objects to store and access data.
- [ ] Followed figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,10 +80,11 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
...
 
     <!-- work one section -->
-
+    <section id="Portfolio">
      <section class="card-works works">
       <div class="image ">
-        <img src="images/Snap.png" alt="Card1"/>
+        <img class="desktop-image" src="images/Snap.png" alt="Project"/>
+        <img class="mobile-image" src="images/Snapshoot1.png" alt="Project"/>
       </div>
```
Comment (andersonlebon) [inline index.html:88]: - [ ]  Kindly comment these cards from your html file and generate your cards dynamically from javascript using an array of objects :+1: 

  **Note**: You can see this [javascript file](https://github.com/PeteUgwu/Portfolio-set-up/pull/11/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3) to get an understanding how to implement both of these requirements:+1:

File: index.html [hunk 2]
```diff
@@ -97,7 +117,8 @@ <h2>Tonic</h2>
 
     <section class="card-works1 works img-right">
       <div class="image-2">
-        <img  src="images/Snap2.png" alt="Card1" />
```
Comment (andersonlebon) [inline index.html:100]: - [ ] Your popup has good content, beautiful pictures, and good colors. But if possible you can improve the design by fixing the size of your images and the buttons, according to the [templet](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143) :+1:

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hello @PeteUgwu 
# Required Changes :recycle: 
omg-oh

:heavy_check_mark:  Descriptive PR Title
:heavy_check_mark:  Descriptive PR Summary
:heavy_check_mark:  Following Github flow
:heavy_check_mark:  Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :grinning: 

### Cheers and Happy coding! 💻 👏
Keep up with the good work

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

___ 

#### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

File: index.html (hunk 1 above)
Comment (andersonlebon) [inline index.html]: - [ ] Your popup has good content, beautiful pictures, and good colors. But if possible you can improve the design by fixing the size of your images and the buttons, according to the [templet](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143) :+1:

File: index.html [hunk 3]
```diff
@@ -61,10 +80,11 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
   
 
     <!-- work one section -->
-
+    <section id="Portfolio">
      <section class="card-works works">
       <div class="image ">
-        <img src="images/Snap.png" alt="Card1"/>
+        <img class="desktop-image" src="images/Snap.png" alt="Project"/>
+        <img class="mobile-image" src="images/Snapshoot1.png" alt="Project"/>
```
Comment (andersonlebon) [inline index.html]: - [ ]  Kindly comment these cards from your html file and generate your cards dynamically from javascript using an array of objects :+1: 

  **Note**: You can see this [javascript file](https://github.com/PeteUgwu/Portfolio-set-up/pull/11/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3) to get an understanding how to implement both of these requirements:+1:

File: index.html (hunk 1 above)
Comment (PeteUgwu) [reply index.html]: Thanks. I have looked and adjusted them

File: index.html (hunk 3 above)
Comment (PeteUgwu) [reply index.html]: Done. thanks for getting me back on track

**Review by alexander16108 — APPROVED** [review]
Hi @PeteUgwu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**